python3/slack_alphabet.py

```diff
@@ -0,0 +1,14 @@
+def _to_letter(c: str) -> str:
+    return f":alphabet-white-{c}:"
+
+
+def slackify(text: str) -> str:
+    return " ".join(map(_to_letter, text))
```

Comment:
Is this going to work correctly when the text contains whitespaces?

Reply:
No idea, got inspired by this: https://github.com/softwaremill/slack-alphabet/blob/master/scala3/src/main/scala/SlackAlphabet.scala which also contains whitespace.

I am ready to remove the spaces if needed :)